google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2186,12 +2229,186 @@ class KmsCryptoKeyVersionIterator(resource_iter_class_factory(
     """The Resource iterator implementation for KMS CryptoKeyVersion."""
 
 
-class KubernetesClusterIterator(resource_iter_class_factory(
-        api_method_name='iter_container_clusters',
-        resource_name='kubernetes_cluster',
-        api_method_arg_key='projectNumber',
-        resource_validation_method_name='container_api_enabled')):
-    """The Resource iterator implementation for Kubernetes Cluster."""
+# class KubernetesClusterIterator(resource_iter_class_factory(
+#         api_method_name='iter_container_clusters',
+#         resource_name='kubernetes_cluster',
+#         api_method_arg_key='projectNumber',
+#         resource_validation_method_name='container_api_enabled')):
+#     """The Resource iterator implementation for Kubernetes Cluster."""
+
+
+class KubernetesClusterIterator(ResourceIterator):
```

Comment:
Why are you re-implementing this class?

Reply:
Deleted.

Reply:
Can you revert back to the version that used the resource_iter_class_factory?